recipes/vmd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "vmd" %}
+{% set version = "1.9.3" %}
+{% set sha256 = "145b4d0cc10b56cadeb71e16c54ab8be713e268f11491714cd617422758ec643" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.ks.uiuc.edu/Research/vmd/vmd-1.9.3/files/final/vmd-{{ version }}.bin.LINUXAMD64-CUDA8-OptiX4-OSPRay111p1.opengl.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  script:
+    - export VMDINSTALLBINDIR=$PREFIX/bin
+    - export VMDINSTALLLIBRARYDIR=$PREFIX/lib
+    - ./configure LINUXAMD64
+    - ./configure
+    - cd src && make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  
+  run:
+    - {{ cdt('mesa-libGL-devel') }}
```

Comment:
The CDTs should only be used for linking and therefore go in the `host` section like this:

```yaml
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
  host:
    - {{ cdt('mesa-libGL-devel') }}
    - xorg-libxi
    - xorg-libxinerama
  run:
    - xorg-libxi
    - xorg-libxinerama
```

This might mean you need then need a `yum_requirements.txt` containing only `mesa-libGL` but try it without first.

Reply:
CI passes after add `mesa-libGL`. Thanks.

Reply:
No problem, LGTM 👍